app/models/course.rb

```diff
@@ -49,7 +49,7 @@ def infer_period_range!
     period =~ /^(\d{4})?/
     year = $1.to_i
 
-    return nil unless year.between? 2014, (DateTime.now.year + 1)
+    return nil unless year.between? 2014, (Time.current.year + 1)
```

Comment:
What is this 2014 supposed to mean? Pre-mumuki times?

Reply:
I guess so!

Reply:
:up: Yes, that is what it means. We used this in recent times to allow inference of course codes and years which where not normalized  - basically because of classroom legacy code.